msgcntr/messageforums-app/src/webapp/jsp/discussionForum/message/dfPendingMessages.jsp

```diff
@@ -44,8 +46,10 @@
 
 			    });
 
-
-
+				var menuLink = $('#forumsQueueMenuLink');
```

Comment:
forumsQueueMenuLink is the span, so you don't need to use closest here. In fact you can just use vanilla:

document.getElementById("forumsQueueMenuLink").classList.add("current");